src/reducers/todos.reducer.js

```diff
@@ -0,0 +1,131 @@
+const initialState = {
+  todoList: [],
+  isLoading: false,
+  errorMessage: '',
+  isSaving: false,
+};
+
+const actions = {
+  fetchTodos: 'fetchTodos',
+  loadTodos: 'loadTodos',
+  setLoadError: 'setLoadError',
+  startRequest: 'startRequest',
+  addTodo: 'addTodo',
+  endRequest: 'endRequest',
+  updateTodo: 'updateTodo',
+  completeTodo: 'completeTodo',
+  revertTodo: 'revertTodo',
+  clearError: 'clearError',
+};
+
+function reducer(state = initialState, action) {
+  switch (action.type) {
+    case actions.fetchTodos:
+      return {
+        ...state,
+        isLoading: true,
+      };
+    case actions.loadTodos:
+      const mappedArray = action.records.map((record) => {
```

Comment:
The naming convention for this data object is far too ambiguous. What is inside a `mappedArray`?